### ISSUE TITLE
Solar Charger load sharing

### ISSUE THREAD

**Issue by soosp** [wagiminator/Power-Boards#9]
Hi,

First of all, thanks for sharing these circuits!
I am mainly interested in the solar charger. As I see it, the battery output feeds directly to the booster circuit. Maybe it would be better to use a P-MOSFET for load sharing, like in LiPo_Charger_LS_TP4056. See considerations [here](https://blog.zakkemble.net/a-lithium-battery-charger-with-load-sharing) and [here](https://www.best-microcontroller-projects.com/tp4056-page2.html).

**Comment by wagiminator** [comment]
Yes, that would undoubtedly be the better solution. However, all of these power boards are just experiments in which I tried out different controller chips. They are not actually intended as final application solutions.
